package.json

```diff
@@ -36,6 +36,7 @@
     "prepare": "husky"
   },
   "devDependencies": {
+    "@prettier/plugin-xml": "^3.4.1",
```

Comment:
I'm not 100% sure this is necessary, but its more of a "better safe then sorry"-approach...

This should maybe help with the pre-commit-hooks etc...

Reply:
Addressed here https://github.com/global-121/121-platform/pull/5659 (it is necessary)